pkg/watcher/skr_webhook_manifest_manager.go

```diff
@@ -87,9 +85,17 @@ func (m *SKRWebhookManifestManager) Install(ctx context.Context, kyma *v1beta2.K
 		return fmt.Errorf("failed to get skrContext: %w", err)
 	}
 
+	_, err = getCertificateSecret(ctx, m.kcpClient, client.ObjectKey{
```

Comment:
you can save a few lines if put this into if condition as `if _, err := getCertificateSecret(...); err != nil`